Codigo/Declares.bas

```diff
@@ -1889,6 +1889,7 @@ Public Type t_UserFlags
     UsandoMacro As Boolean
     PendienteDelSacrificio As Byte
     PendienteDelExperto As Byte
+    TalismanDeLosMuertos As Byte
```

Comment:
can you move this to Status mask and create a value in e_StatusMask and use english for the definition?